utils.py

```diff
@@ -4,6 +4,7 @@
 from os.path import isfile, join
 import zipfile
 import json
```

Comment:
Rename InstalledSoftware1. what's the 1? also, file names in python should be in camel_case (installed_software)